the_well/data/utils.py

```diff
@@ -31,3 +41,32 @@ def preprocess_batch(
         }
     },
 }
+
+
+def is_dataset_in_the_well(dataset_name: str) -> bool:
+    """Tell whether a dataset is in the Well or not.
+    Accept `dummy` as a valid dataset.
+
+    Args:
+        dataset_name: The name of the dataset.
+
+    Returns:
+        True if the dataset is in the Well, False otherwise.
+    """
+    is_valid = dataset_name in WELL_DATASETS or dataset_name == "dummy"
+    return is_valid
```

Comment:
Do we still need dummy?

Reply:
Dummy is still used in the tests and relevant to other projects. Let's keep it for now but it's good idea to remove it in the future.